pkgpanda/docs/package_concepts.md

```diff
@@ -1,14 +1,14 @@
 # Package concepts
 
-* Package Name *
+*Package Name*
 
 The name which other packages will know this package by and use. Package names must be valid Linux folder names, should
 be case insensitive most often lower case only. Valid characters are `[a-zA-Z0-9@._+-]`. They may not start with a hyphen
 or a dot. Must be at least one character long. A package name may not contain '--'.
 
 *Package ID*
 
-`name--id` combination package name + unique, reproducible package hash
+`name--id` combination package name + arbitrary information (most often a version indicator). The packaging system needs to extract the package name from a package id. Valid characters are `[a-zA-Z0-9@._+-]`
```

Comment:
`name--version`. That's the terminology used within pkgpanda itself, and it's confusing to tell someone that the package ID contains an ID.